### PR TITLE
feature/fingerbank_settings_api

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Fingerbank Settings API

Delete branch after merge
=========================
NO

Checklist
=========
- [ ] Document the feature
- [x] Add unit tests
- [ ] Add acceptance tests (TestLink)

### CODE REVIEW

File: lib/pf/ConfigStore/FingerbankSettings.pm [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+=head2 remove
+
+Delete an existing item
+
+=cut
+
+sub remove { return; }
+
+=head2 cleanupAfterRead
```
Comment (julsemaan) [inline lib/pf/ConfigStore/FingerbankSettings.pm]: Doesn't belong here anymore

File: lib/pf/UnifiedApi.pm [hunk 2]
```diff
@@ -1155,6 +1156,26 @@ sub setup_api_v1_config_filters_routes {
...
+
+sub setup_api_v1_config_fingerbank_settings_routes {
+    my ($self, $root) = @_;
+    my ($collection_route, $resource_route) =
+      $self->setup_api_v1_std_config_routes(
+        $root,
+        "Config::FingerbankSettings",
+        "/fingerbank_settings",
+        "/fingerbank_setting/#fingerbank_setting_id",
+        "api.v1.Config.SyslogParsers"
```
Comment (julsemaan) [inline lib/pf/UnifiedApi.pm]: Copy/paste error

File: lib/pfconfig/namespaces/config/FingerbankDoc.pm [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+pfconfig::namespaces::config::FingerbankDoc
+
+=cut
+
+=head1 DESCRIPTION
+
+pfconfig::namespaces::config::FingerbankDoc
+
+This module creates the configuration hash associated to documentation.conf
```
Comment (julsemaan) [inline lib/pfconfig/namespaces/config/FingerbankDoc.pm]: wrong file name

**Comment by jrouzierinverse** [comment]
done

**Pull request merged**